docs/standard/terminology.md

```diff
@@ -0,0 +1,69 @@
+# Terminology
+
+In order to use key terms consistently across the standard documentation and OCP communication in general, terminology is managed according to a process that ensures both quality and reactivity.
+
+![Terminology process overview](https://www.lucidchart.com/invitations/accept/2eee45ed-5e62-4636-9a44-24b4414f273f)
+
+## 1. Proposition and validation
+
+When new content is added to the documentation and new concepts are introduced in the standard, new terms may need to be translated.
+
+**Reminder:** a term is a word or group of words that must be used and translated consistently to ensure a good understanding and usage of the content. Synonyms of terms must consequently be banned. Expressions that are not terms are called *generic* expressions.
+
+| #   | Step name   | Description                                                                                          | Tool               |
+| --- | ----------- | ---------------------------------------------------------------------------------------------------- | ------------------ |
+| 1   | Proposition | The writer of the standard writes down a list of words or groups of words they consider being terms. | Github issue       |
+| 2   | Validation  | A terminologist validates that the proposed expressions are actually terms.                          | Github issue       |
+| 3   | Insertion   | The new terms are added in the working document, checking for duplicates                .            | Google spreadsheet |
+
+
+## 2. Definition
+
+When new terms are added to the working document, subject matter experts (SME) in the source language are noticed, and write a definition for each term. The definitions must be bound to the context of the source documents, and not general.
+
+Using other terms of the glossary in the definitions is a good practice and helps contextualizing the terms.
+
+If the definition is copied from an existing publication, please add the URL or reference at the end of the definition.
+
+| #   | Step name  | Description                                                                           | Tool               |
+| --- | ---------- | ------------------------------------------------------------------------------------- | ------------------ |
+| 1   | Definition | The source language SME writes a definition for each term, or copies an existing one. | Google spreadsheet |
+
+
+
+## 3. Translation
+
+When the definition of the new terms are ready, subject matter experts (SME) are noticed. They only translate terms to their native language.
```

Comment:
definition -> definitions
"subject matter experts (SME) are noticed" -> "subject matter experts (SME) in the other languages are notified"

Reply:
OK